app/models/lesson.rb

```diff
@@ -26,6 +26,12 @@ def import_content_from_github
     LessonContentImporter.for(self)
   end
 
+  def display_title
```

Comment:
By moving the display title to the lesson, we can display the same title all over. In admin, the react components and lessons. We normally shouldn't have display logic in the model, but it's a worthwhile trade off for now.

In the future we may have separate models for lessons and projects, but there isn't enough differences in them to make it worth it doing that yet.

Reply:
I think there's an argument that the prefix is part of the domain rather than purely display logic. Agree that this is a good tradeoff in current state